runelite-client/src/main/java/net/runelite/client/plugins/worldmap/WorldMapConfig.java

```diff
@@ -152,4 +152,26 @@ default boolean miscellaneousTeleportIcon()
 	{
 		return true;
 	}
+
+	@ConfigItem(
+		keyName = WorldMapPlugin.CONFIG_KEY_QUEST_START_ICON,
+		name = "Show quest start level requirements",
+		description = "Indicates quests you don't have the requirements to start",
+		position = 12
+	)
+	default boolean questStartIcon()
+	{
+		return true;
+	}
+
+	@ConfigItem(
+		keyName = WorldMapPlugin.CONFIG_KEY_QUEST_START_TOOLTIPS,
+		name = "Show quest names",
```

Comment:
I would merge these 2 toggles to 1, i feel like having both is overdone. Just name it Show quest names and requirements